loadJsonFile.py

```diff
@@ -0,0 +1,60 @@
+import googlemaps
+import json
+import requests
+
+API_KEY = 'AIzaSyDFtKtm0jnjckE-lB0LkO3SKlWmQIJush8';
+gmaps = googlemaps.Client(key=API_KEY)
+
+# Load file -> load 'result.json'
+with open('result.json') as f:
+    read_data = f.read()
+
+# Dict type is 'list'
+datainfo = json.loads(read_data)
+
+data = dict()
+
+# If the information is unknown, save 'unknown'
+for item in datainfo:
+    try: data['name'] = item['name']
+    except KeyError: data['name'] = ''
+    try: data['address'] = item['formatted_address']
+    except KeyError: data['address'] = ''
+    try: data['latitude'] = item['geometry']['location']['lat'] 
+    except KeyError: data['latitude'] = '' 
+    try: data['longitude'] = item['geometry']['location']['lng']
+    except KeyError: data['longitude'] = ''  
+    try: data['phone'] = item['formatted_phone_number'] 
+    except KeyError: data['phone'] = '' 
+    try: data['rating'] = item['rating'] 
+    except KeyError: data['rating'] = '' 
+    try: data['website'] = item['website']
+    except KeyError: data['website'] = ''
+
+    opendatalist = []
```

Comment:
Are we using camel case for variable naming? I thought I left a comment a while ago saying you guys should decide on this.